<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testng-results ignored="0" total="3" passed="3" failed="0" skipped="0">
=======
<testng-results ignored="0" total="7" passed="2" failed="5" skipped="0">
>>>>>>> branch 'master' of https://github.com/VigneshDhakshnamoorthy/Automation.git
  <reporter-output>
  </reporter-output>
<<<<<<< HEAD
  <suite started-at="2021-06-30T14:13:32 IST" name="Default suite" finished-at="2021-06-30T14:13:35 IST" duration-ms="3438">
=======
  <suite started-at="2021-06-10T16:43:53 IST" name="Default suite" finished-at="2021-06-10T16:43:54 IST" duration-ms="386">
>>>>>>> branch 'master' of https://github.com/VigneshDhakshnamoorthy/Automation.git
    <groups>
      <group name="success">
        <method signature="TestcasesTest.testtopassTest1(java.lang.String)[pri:1, instance:testng.sample.TestcasesTest@a2431d0]" name="testtopassTest1" class="testng.sample.TestcasesTest"/>
        <method signature="TestcasesTest.testtopassTest2(java.lang.String)[pri:1, instance:testng.sample.TestcasesTest@a2431d0]" name="testtopassTest2" class="testng.sample.TestcasesTest"/>
        <method signature="TestcasesTest.testtopassTest3(java.lang.String)[pri:1, instance:testng.sample.TestcasesTest@a2431d0]" name="testtopassTest3" class="testng.sample.TestcasesTest"/>
        <method signature="TestcasesTest.testtopassTest4(java.lang.String)[pri:2, instance:testng.sample.TestcasesTest@a2431d0]" name="testtopassTest4" class="testng.sample.TestcasesTest"/>
      </group> <!-- success -->
      <group name="Fail">
        <method signature="TestcasesTest.testtofailTest1(java.lang.String)[pri:3, instance:testng.sample.TestcasesTest@a2431d0]" name="testtofailTest1" class="testng.sample.TestcasesTest"/>
        <method signature="TestcasesTest.testtofailTest2(java.lang.String)[pri:3, instance:testng.sample.TestcasesTest@a2431d0]" name="testtofailTest2" class="testng.sample.TestcasesTest"/>
      </group> <!-- Fail -->
    </groups>
<<<<<<< HEAD
    <test started-at="2021-06-30T14:13:32 IST" name="Default test" finished-at="2021-06-30T14:13:35 IST" duration-ms="3438">
      <class name="api.samples.api_test_1">
        <test-method signature="rest_api_2()[pri:1, instance:api.samples.api_test_1@fa36558]" started-at="2021-06-30T14:13:32 IST" name="rest_api_2" finished-at="2021-06-30T14:13:35 IST" duration-ms="3012" status="PASS">
=======
    <test started-at="2021-06-10T16:43:53 IST" name="Default test" finished-at="2021-06-10T16:43:54 IST" duration-ms="386">
      <class name="testng.sample.TestcasesTest">
        <test-method signature="testtopassTest1(java.lang.String)[pri:1, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtopassTest1" finished-at="2021-06-10T16:43:54 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'success' is required by @Test on method testtopassTest1 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'success' is required by @Test on method testtopassTest1 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
>>>>>>> branch 'master' of https://github.com/VigneshDhakshnamoorthy/Automation.git
          <reporter-output>
          </reporter-output>
<<<<<<< HEAD
        </test-method> <!-- rest_api_2 -->
        <test-method signature="rest_api_1()[pri:2, instance:api.samples.api_test_1@fa36558]" started-at="2021-06-30T14:13:35 IST" name="rest_api_1" finished-at="2021-06-30T14:13:35 IST" duration-ms="117" status="PASS">
=======
        </test-method> <!-- testtopassTest1 -->
        <test-method signature="testtopassTest2(java.lang.String)[pri:1, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtopassTest2" finished-at="2021-06-10T16:43:54 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'success' is required by @Test on method testtopassTest2 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'success' is required by @Test on method testtopassTest2 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
>>>>>>> branch 'master' of https://github.com/VigneshDhakshnamoorthy/Automation.git
          <reporter-output>
          </reporter-output>
<<<<<<< HEAD
        </test-method> <!-- rest_api_1 -->
        <test-method signature="rest_api_3()[pri:3, instance:api.samples.api_test_1@fa36558]" started-at="2021-06-30T14:13:35 IST" name="rest_api_3" finished-at="2021-06-30T14:13:35 IST" duration-ms="261" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rest_api_3 -->
      </class> <!-- api.samples.api_test_1 -->
=======
        </test-method> <!-- testtopassTest2 -->
        <test-method signature="testtopassTest3(java.lang.String)[pri:1, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtopassTest3" finished-at="2021-06-10T16:43:54 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'success' is required by @Test on method testtopassTest3 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'success' is required by @Test on method testtopassTest3 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testtopassTest3 -->
        <test-method signature="testtopassTest4(java.lang.String)[pri:2, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtopassTest4" data-provider="dp_name" finished-at="2021-06-10T16:43:54 IST" duration-ms="51" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[DP_1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testtopassTest4 -->
        <test-method signature="testtopassTest4(java.lang.String)[pri:2, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtopassTest4" data-provider="dp_name" finished-at="2021-06-10T16:43:54 IST" duration-ms="13" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[DP_2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testtopassTest4 -->
        <test-method signature="testtofailTest1(java.lang.String)[pri:3, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtofailTest1" finished-at="2021-06-10T16:43:54 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'Fail' is required by @Test on method testtofailTest1 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'Fail' is required by @Test on method testtofailTest1 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testtofailTest1 -->
        <test-method signature="testtofailTest2(java.lang.String)[pri:3, instance:testng.sample.TestcasesTest@a2431d0]" started-at="2021-06-10T16:43:54 IST" name="testtofailTest2" finished-at="2021-06-10T16:43:54 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'Fail' is required by @Test on method testtofailTest2 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'Fail' is required by @Test on method testtofailTest2 but has not been marked @Optional or defined
in C:\Users\Vignesh\AppData\Local\Temp\testng-eclipse-1713057668\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testtofailTest2 -->
      </class> <!-- testng.sample.TestcasesTest -->
>>>>>>> branch 'master' of https://github.com/VigneshDhakshnamoorthy/Automation.git
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
